accurest-core/src/main/groovy/io/codearte/accurest/dsl/WiremockRequestStubStrategy.groovy

```diff
@@ -27,12 +30,47 @@ class WiremockRequestStubStrategy extends BaseWiremockStubStrategy {
 	private Map<String, Object> buildRequestContent(ClientRequest request) {
 		return ([method    : request?.method?.clientValue,
 		        headers   : buildClientRequestHeadersSection(request.headers)
-		] << appendUrl(request) << appendBody(request)).findAll { it.value }
+		] << appendUrl(request) << appendQueryParameters(request) << appendBody(request)).findAll { it.value }
 	}
 
 	private Map<String, Object> appendUrl(ClientRequest clientRequest) {
-		Object url = clientRequest?.url?.clientValue
-		return url instanceof Pattern ? [urlPattern: ((Pattern)url).pattern()] : [url: url]
+		def urlPath = clientRequest?.urlPath?.clientValue
+		if (urlPath) {
+			return [urlPath: urlPath]
+		}
+		def url = clientRequest?.url?.clientValue
+		return url instanceof Pattern ? [urlPattern: url.pattern()] : [url: url]
+	}
+
+	private Map<String, Object> appendQueryParameters(ClientRequest clientRequest) {
+		def queryParameters = clientRequest?.urlPath?.queryParameters ?: clientRequest?.url?.queryParameters
```

Comment:
I don't remember clearly, but I think it was some other case.

Unfortunately null-safe operator in Groovy can be misleading and repeating `?.` seems to be needed :(.

See: https://groovyconsole.appspot.com/script/5734309889572864 and http://www.eclecticlogic.com/2015/02/17/groovy-oddities/ (the first paragraph).